### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by sanamrahimi**
## in this pull request, I have implemented features that were mentioned in the requirement.
# Requirements

- Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- Create a new GitHub repo.
- Create a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
- Add a .gitignore file.
- Add node_modules to gitignore file.
- Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
- Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- Add 3 files (each one in a separate commit):
- A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
- A .css file that includes one class with styles for your header (make it in your favorite color).
- A test.md file with the text "This file should be ignored by git".
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,287 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [hello microverse] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] Kindly consider removing these default statements. Removing them will help us have a professional readme file ☺️.

File: README.md [hunk 2]
```diff
@@ -1 +1,287 @@
...
+
+**[hello microverse]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] Kindly consider removing these default statements. Removing them will help us have a professional readme file ☺️.

File: README.md [hunk 3]
```diff
@@ -1 +1,287 @@
...
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] Kindly consider removing these default statements. Removing them will help us have a professional readme file ☺️.

File: README.md [hunk 4]
```diff
@@ -1 +1,287 @@
...
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] Kindly consider removing these default statements. Removing them will help us have a professional readme file ☺️.

File: README.md [hunk 5]
```diff
@@ -1 +1,287 @@
...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] I have noticed that your `key features` is not customized. Kindly ensure to add your project's key features. This helps us to have a professional README file ☺️.

File: README.md [hunk 6]
```diff
@@ -1 +1,287 @@
...
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] I have noticed that your `future features` is not customized. Kindly ensure to add your project's future features. This helps us to have a professional README file ☺️.
- [ ] Kindly consider removing these default statements. Removing them will help us have a professional readme file ☺️.

File: README.md [hunk 7]
```diff
@@ -1 +1,287 @@
...
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] [OPTIONAL] I have noticed that your `FAQ` is not customized. Kindly ensure to add your project's FAQ. This helps us to have a professional README file ☺️.

File: README.md [hunk 8]
```diff
@@ -1 +1,287 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] I have noticed that your license link is broken. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose. At this stage, it means your project is not yet licensed and hence the code can not be reused. Kindly ensure to add the license file to your project directory ☺️.

File: README.md [hunk 9]
```diff
@@ -1 +1,287 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] I have noticed that your `support subsection` is not customized. Kindly ensure to add your project's support section. This helps us to have a professional README file ☺️. It also helps to increase the project engagement. In addition, kindly consider removing these default statements. Removing them will help us have a professional readme file ☺️.

File: README.md [hunk 10]
```diff
@@ -1 +1,287 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] I have noticed that your `acknowledgments` is not customized. Kindly ensure to add your project's acknowledgments. This helps us to have a professional README file ☺️. In addition, kindly consider removing these default statements. Removing them will help us have a professional readme file ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @sanamrahimi 🙋‍♂️
Good job working on your hello Microverse project.  

You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

# Required Changes :recycle: 
- _Kindly consider the inline review comments._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: README.md [hunk 11]
```diff
@@ -1 +1,163 @@
...
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```
Comment (victorokoroji) [inline README.md:31]: - [ ] Please note that the MIT license gives users 🧑‍🦱 express permission to reuse code for any purpose. At this stage, it means your project is not yet licensed and hence the code can not be reused. Kindly ensure to add the license file to your project directory as the previous reviewer suggested.

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @sanamrahimi, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- I will suggest you use a more descriptive branch name that better describes what changes or features you are working on. Your branch name should be the bane of a feature and not a feature itself


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @sanamrahimi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**